src/ontology/cro-edit.owl

```diff
@@ -1,8 +1,9 @@
-Prefix(:=<http://purl.obolibrary.org/obo/cro.owl#>)
+Prefix(:=<http://purl.obolibrary.org/obo/cro.owl/>)
 Prefix(owl:=<http://www.w3.org/2002/07/owl#>)
 Prefix(rdf:=<http://www.w3.org/1999/02/22-rdf-syntax-ns#>)
 Prefix(xml:=<http://www.w3.org/XML/1998/namespace>)
 Prefix(xsd:=<http://www.w3.org/2001/XMLSchema#>)
+Prefix(obda:=<https://w3id.org/obda/vocabulary#>)
```

Comment:
I think that Ontop OBDA Plugin is adding extraneous things to the stuff you're editing in Protege...